cmd/gameserver/packet/inventory.go

```diff
@@ -0,0 +1,130 @@
+package packet
+
+import (
+	"github.com/ubis/Freya/share/log"
+	"github.com/ubis/Freya/share/models/character"
+	"github.com/ubis/Freya/share/network"
+	"github.com/ubis/Freya/share/rpc"
+)
+
+func SendStorageExchangeMoveStatusPacket(session *network.Session, result byte) {
+	packet := network.NewWriter(STORAGE_EXCHANGE_MOVE)
```

Comment:
I'm not really fan of long function names, perhaps we should name it into `NotifyStorageExhange`?

Same thing with `SendStorageExchangeMoveMasterServerPacket` -> `UpdatePlayerStorage`?